website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
Hm, is there any reason we can't use just the first regex, `/(^\/)|(\/$)/g`, so that we always strip leading and trailing slashes (if they exist)?

That way, we only need one argument and this function becomes simpler.